assets/less/cds-rdm/globals/mixins.less

```diff
@@ -0,0 +1,614 @@
+.flat-shadow-orange(){
```

Comment:
Please prefer to embed this in the background image, instead of CSS. This is not maintainable.

Reply:
I will replace it with a less function later on, I didn't have time to create a proper one but I will do it gradually. I agree it is not easy to maintain. If you don't mind I will leave it temporarily here

Reply:
Ok, but for me they should be part of the background image, and not as extra CSS.